website/docs/basics/glossary.md

```diff
@@ -476,10 +479,19 @@ The sender of a transaction often includes a fee to the network for processing t
 
 #### See Also
 - [Shannon](#shannon)
+- [Fee Rate](#Fee-Rate)
 - [Fee rate in RFC on Nervos Network Github](https://github.com/nervosnetwork/ckb/tree/develop/rpc#error-poolrejectedtransactionbyminfeerate)
 
 ---
 
+### Fee Rate
+If at the same size, the amount of fee a miner can get when include a transaction.
```

Comment:
A tip that the user offers to the miner to prioritize their transaction over others.

Reply:
It's more concise than the old one, but both are more likely the concept `Fee`  than `Feerate`